Source/Renderer/VertexArray.js

```diff
@@ -308,6 +309,10 @@ define([
                 hasInstancedAttributes = true;
                 break;
             }
+            if (defined(vaAttributes[i].value)) {
+                hasConstantAttributes = true;
+                break;
```

Comment:
Are you sure this `break` statement and the one above should be here?

Either split this into a separate `for` loop or merge into the above and remove all the `breaks`.